### ISSUE TITLE
Focus is not visible on Pivot with link format as tabs in high contrast

### ISSUE THREAD

**Issue by bhaveshbhati** [OfficeDev/office-ui-fabric-react#3735]
### Bug Report

- __Package version(s)__: Latest
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: (Blocking, High, Normal, Low) 

Products/sites affected: (if applicable)

#### Describe the issue:

Focus is not visible on selected pivot tab item in Pivot (linkformat = PivotLinkFormat.tabs ) in high contrast mode(Black).

In https://developer.microsoft.com/en-us/fabric#/components/pivot, select Bar pivot. Now go to Foo via keyboard and then try to come back to Bar via keyboard.

#### Actual behavior:

Focus is not visible.

![image](https://user-images.githubusercontent.com/1016447/35042397-82dd34dc-fbae-11e7-8b5d-cd65804e7921.png)

#### Expected behavior:

Focus is visible.

### If applicable, please provide a codepen repro:

https://developer.microsoft.com/en-us/fabric#/components/pivot

**Comment by betrue-final-final** [comment]
It should look like this:

![image](https://user-images.githubusercontent.com/20363037/39158367-836797a6-4714-11e8-82ec-5424e500cdb5.png)
